packages/astro/src/core/ignore-files.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'node:fs';
+import { joinPaths } from './path.js';
+
+/**
+ * use .gitignore file to ignore files
+ * @param root astro root path
+ */
+export async function ignoreFiles(root: string) {
```

Comment:
We could call this function `getGitIgnorePaths`, which is more in line with what the function does

Reply:
fixed

name: getGitignorePaths
file: https://github.com/guobin211/astro/blob/main/packages/astro/src/core/ignore-files.ts